src/main/java/com/stripe/model/Price.java

```diff
@@ -0,0 +1,328 @@
+  /** Whether the price can be used for new purchases. */
+  @SerializedName("active")
+  Boolean active;
+
+  /**
+   * Describes how to compute the price per period. Either {@code per_unit} or {@code tiered}.
+   * {@code per_unit} indicates that the fixed amount (specified in {@code unit_amount} or {@code
+   * unit_amount_decimal}) will be charged per unit in {@code quantity} (for prices with {@code
+   * usage_type=licensed}), or per unit of total usage (for prices with {@code usage_type=metered}).
+   * {@code tiered} indicates that the unit pricing will be computed using a tiering strategy as
+   * defined using the {@code tiers} and {@code tiers_mode} attributes.
+   *
+   * <p>One of {@code per_unit}, or {@code tiered}.
+   */
+  @SerializedName("billing_scheme")
+  String billingScheme;
+
+  /** Time at which the object was created. Measured in seconds since the Unix epoch. */
+  @SerializedName("created")
+  Long created;
+
+  /**
+   * Three-letter <a href="https://www.iso.org/iso-4217-currency-codes.html">ISO currency code</a>,
+   * in lowercase. Must be a <a href="https://stripe.com/docs/currencies">supported currency</a>.
+   */
+  @SerializedName("currency")
+  String currency;
+
+  /** Always true for a deleted object. */
+  @SerializedName("deleted")
+  Boolean deleted;
+
+  /** Unique identifier for the object. */
+  @Getter(onMethod_ = {@Override})
+  @SerializedName("id")
+  String id;
+
+  /**
+   * Has the value {@code true} if the object exists in live mode or the value {@code false} if the
+   * object exists in test mode.
+   */
+  @SerializedName("livemode")
+  Boolean livemode;
+
+  /** A lookup key used to retrieve prices dynamically from a static string. */
+  @SerializedName("lookup_key")
+  String lookupKey;
+
+  /**
+   * Set of key-value pairs that you can attach to an object. This can be useful for storing
+   * additional information about the object in a structured format.
+   */
+  @Getter(onMethod_ = {@Override})
+  @SerializedName("metadata")
+  Map<String, String> metadata;
+
+  /** A brief description of the plan, hidden from customers. */
+  @SerializedName("nickname")
+  String nickname;
+
+  /**
+   * String representing the object's type. Objects of the same type share the same value.
+   *
+   * <p>Equal to {@code price}.
+   */
+  @SerializedName("object")
+  String object;
+
+  /** The ID of the product this price is associated with. */
+  @SerializedName("product")
+  @Getter(lombok.AccessLevel.NONE)
+  @Setter(lombok.AccessLevel.NONE)
+  ExpandableField<Product> product;
+
+  /** The recurring components of a price such as {@code interval} and {@code usage_type}. */
+  @SerializedName("recurring")
+  Recurring recurring;
+
+  /**
+   * Each element represents a pricing tier. This parameter requires {@code billing_scheme} to be
+   * set to {@code tiered}. See also the documentation for {@code billing_scheme}.
+   */
+  @SerializedName("tiers")
+  List<Plan.Tier> tiers;
```

Comment:
This is problematic and we need to fix it internally, possibly by duplicating the class/resource. `Price` is a new resource and it shouldn't reference the deprecated one. And we can't change `Plan` to use the `Price.Tier` either since it's a breaking change.

Reply:
@areagle-stripe This one still seems to be incorrect.